docs/NativeModulesAndroid.md

```diff
@@ -89,21 +89,43 @@ ReadableArray -> Array
 ### Register the Module
 
 The last step within Java is to register the Module; this happens in the `createNativeModules` of your apps package. If a module is not registered it will not be available from JavaScript.
-
+In the Module registry class it is important you override three methods `createNativeModules`, `createJSModules` and `createViewManagers`.
 ```java
-class AnExampleReactPackage implements ReactPackage {
+//AnExampleReactPackage.java
+package com.facebook.react.modules.toast;
 
-  ...
+import java.util.ArrayList;
```

Comment:
With the comments below, can remove this import (import java.util.Arrays and java.util.Collections) instead.